src/chainer.js

```diff
@@ -12,7 +13,7 @@ function chainer(functions) {
     }
```

Comment:
The logic inside 'chainer' function is not complete. There is no implementation of applying the given functions in a left-to-right order on the input.